test/util/MessagesSpec.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package util
+
+import helpers.TestHelper
+import play.api.i18n.Messages
+import play.api.i18n.Messages.MessageSource
+
+import scala.io.Source
+
+class MessagesSpec extends TestHelper {
+
+  private val MatchSingleQuoteOnly = """\w+'{1}\w+""".r
+  private val MatchBacktickQuoteOnly = """`+""".r
+  private val MatchForwardTickQuoteOnly = """’+""".r
+
+  private val englishMessages = parseMessages("conf/messages")
```

Comment:
Might be a nitpick but I think the standard is to name the two messages files, 'messages.cy' and 'messages.en'